### PR TITLE
Enable sp build web in styling

### PR DESCRIPTION
**Pull request by srideshpande**
#### Pull request checklist

- [ ] Added a manifest file to styling package
- [ ] Enabled sp-build-web in styling package

### CODE REVIEW

**Comment by dzearing** [comment]
looks like you ran rush generate with a default registry defined in your .npmrc. First, check your user-scoped .npmrc and remove that if it's there.

Then run `npm cache clean`.

Then `rush generate`.

File: src/components/Dropdown/Dropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,248 @@
+import * as React from 'react';
+import { IDropdownProps, IDropdownOption } from './Dropdown.Props';
+import { css } from '../../utilities/css';
+import { EventGroup } from '../../utilities/eventGroup/EventGroup';
+import { findIndex } from '../../utilities/array';
+import { KeyCodes } from '../../utilities/KeyCodes';
+import { getId } from '../../utilities/object';
```
Comment (dzearing) [inline src/components/Dropdown/Dropdown.tsx]: you didn't intend to include this right?

File: packages/styling/gulpfile.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ build.sass.isEnabled = () => false;
 build.karma.isEnabled = () => false;
 build.text.isEnabled = () => false;
 build.preCopy.isEnabled = () => false;
-build.webpack.isEnabled = () => false;
+// build.webpack.isEnabled = () => false;
```
Comment (dzearing) [inline packages/styling/gulpfile.js]: just delete the unneeded line.

File: packages/styling/gulpfile.js [hunk 3]
```diff
@@ -1,9 +1,9 @@
 'use strict';
 
-let build = require('@microsoft/web-library-build');
+let build = require('@microsoft/sp-build-web');
 let gulp = require('gulp');
 
-// Configure TypeScript.
+// Configure TypeScript 2.0.
```
Comment (dzearing) [inline packages/styling/gulpfile.js]: no need to add the 2.0

File: src/components/Dropdown/Dropdown.tsx (hunk 1 above)
Comment (srideshpande) [inline src/components/Dropdown/Dropdown.tsx]: Yeah.

File: packages/styling/gulpfile.js (hunk 3 above)
Comment (srideshpande) [inline packages/styling/gulpfile.js]: done!

**Pull request merged**